src/components/SignIn/SignIn.js

```diff
@@ -63,7 +65,15 @@ class SignIn extends Component {
 
     submitHandler = () => {
         if (this.props.formIsValid) {
-            this.props.onSubmit(true);
+            
+
+            //this.props.onSubmit(true);
+
+            /*Tests here*/
```

Comment:
what tests?

Reply:
Component is not needed anymore. It has been removed.